### ISSUE TITLE
AttributeError: 'QgsMeshDataProvider' object has no attribute 'identify'

### ISSUE THREAD

**Issue by AlisterH** [jorgealmerio/valuetool#7]
Using this in QGIS 3.14.0 and 3.12.x with a HEC-RAS mesh layer produces an error:
```
2020-06-26T19:12:34     WARNING    Traceback (most recent call last):
              File "C:/Users/alistair/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\valuetool\valuewidget.py", line 393, in printValue
              ident = layer.dataProvider().identify(pos, QgsRaster.IdentifyFormatValue, canvas.extent(), width, height ).results()
             AttributeError: 'QgsMeshDataProvider' object has no attribute 'identify'
```
Should I be trying in an older version to see if that makes a difference?
Perhaps it just isn't supported by all types of mesh layer?

**Comment by jorgealmerio** [comment]
Could you please send me this mesh layer or part of it for me to reproduce the error?

**Comment by AlisterH** [comment]
https://civilplan-my.sharepoint.com/:u:/g/personal/alister_civilplan_co_nz/EdyS7KdZlIRAslTQcAfKOWQBXFFjDPIuQgq78JXIpNw_Jw?e=RtLodY

**Comment by jorgealmerio** [comment]
I hope it solved on uploaded version 3.0.7!
Let me know if not solved!
